metadata-editor/conf/routes

```diff
@@ -20,6 +20,12 @@ GET     /metadata/:id/usage-rights                      controllers.EditsControl
 PUT     /metadata/:id/usage-rights                      controllers.EditsController.setUsageRights(id: String)
 DELETE  /metadata/:id/usage-rights                      controllers.EditsController.deleteUsageRights(id: String)
 
+# Collections
+GET     /collections                                    controllers.CollectionsController.getCollections
+POST    /collections                                    controllers.CollectionsController.addCollectionFromJson
+PUT     /collections/*collection                        controllers.CollectionsController.addCollectionFromString(collection: String)
```

Comment:
The `PUT` here is idempotent, except for updating the actionData due to [the way we add things](https://github.com/guardian/grid/pull/1479/files#diff-54243d7bda00027741bfd38375de1575R18). We could make a read endpoint to complete the puzzle, with little use though. 

I'm not sure I like it as it's nicer to get an actual list from the client that we then create the ID from, so I'm going to stick with the `POST` as we don't know what the ID is until we've made the collection.